pkg/security/probe/probe_linux.go

```diff
@@ -309,16 +309,15 @@ func (p *Probe) PlaySnapshot() {
 	// Get the snapshotted data
 	var events []*model.Event
 
-	entryToEvent := func(entry *model.ProcessCacheEntry) {
+	entryToEvent := func(entry *model.ProcessContext) {
 		if entry.Source != model.ProcessCacheEntryFromSnapshot {
 			return
 		}
-		entry.Retain()
```

Comment:
how to we ensure that the ProcessCacheEntry holding the ProcessContext is not going to be released and reuse if we don't retain it ?